src/components/shared/formItems/EthAddress.tsx

```diff
@@ -24,65 +27,49 @@ export default function EthAddress({
     theme: { colors },
   } = useContext(ThemeContext)
 
-  const [addressForENS, setAddressForENS] = useState<string>()
-  const [displayValue, setDisplayValue] = useState<string>()
+  const [value, setValue] = useState<string>()
+  const [addressForENSName, setAddressForENSName] = useState<string>()
+  const [ENSName, setENSName] = useState<string>()
```

Comment:
think naming should always be consistent, so always using camel for vars - bigger fan of `ensName` rather than `ENSName`, thoughts?

Reply:
Yes totally! good call